### PR TITLE
statistics: use `[]float64` to represent store loads

### PR DESCRIPTION
**Pull request by disksing**
### What problem does this PR solve?
Currently, all store loads are saved in different variables and accessed using different functions.
Too many duplicated code and hard to extend.

### What is changed and how it works?
Save loads using `[]float64` and access them using enums.

### Check List

Tests
- Unit test

### Release note
- No release note

### CODE REVIEW

File: server/statistics/store_collection.go [hunk 1]
```diff
@@ -108,23 +108,19 @@ func (s *storeStatistics) Observe(store *core.StoreInfo, stats *StoresStats) {
...
-	storeCPUUsage := stats.GetStoreCPUUsage(store.GetID())
-	storeStatusGauge.WithLabelValues(storeAddress, id, "store_cpu_usage").Set(storeCPUUsage)
-	storeDiskReadRate := stats.GetStoreDiskReadRate(store.GetID())
-	storeStatusGauge.WithLabelValues(storeAddress, id, "store_disk_read_rate").Set(storeDiskReadRate)
-	storeDiskWriteRate := stats.GetStoreDiskWriteRate(store.GetID())
-	storeStatusGauge.WithLabelValues(storeAddress, id, "store_disk_write_rate").Set(storeDiskWriteRate)
+
+	storeStatusGauge.WithLabelValues(storeAddress, id, "store_write_rate_bytes").Set(storeFlowStats.GetLoad(StoreWriteBytes))
+	storeStatusGauge.WithLabelValues(storeAddress, id, "store_read_rate_bytes").Set(storeFlowStats.GetLoad(StoreReadBytes))
+	storeStatusGauge.WithLabelValues(storeAddress, id, "store_write_rate_keys").Set(storeFlowStats.GetLoad(StoreWriteKeys))
```
Comment (lhy1024) [inline server/statistics/store_collection.go:114]: without readkey

File: server/statistics/store.go [hunk 2]
```diff
@@ -79,187 +77,36 @@ func (s *StoresStats) Set(storeID uint64, stats *pdpb.StoreStats) {
...
-	for _, store := range ss {
-		if store.IsUp() {
-			stats, ok := s.rollingStoresStats[store.GetID()]
-			if !ok {
-				continue
-			}
-			writeRate, readRate = stats.GetKeysRate()
-			totalKeysWriteRate += writeRate
-			totalKeysReadRate += readRate
+	for _, store := range stores {
```
Comment (lhy1024) [inline server/statistics/store.go:87]: why to set zero

**Review by lhy1024 — COMMENTED** [review]
the rest LGTM

File: server/statistics/store_collection.go (hunk 1 above)
Comment (disksing) [reply server/statistics/store_collection.go:114]: next line is read keys

File: server/statistics/store.go (hunk 2 above)
Comment (disksing) [reply server/statistics/store.go:87]: To reuse the map. Don't have to allocate new map every time.

**Review by rleungx — APPROVED** [review]

**Comment by disksing** [comment]
@lhy1024 come a LGTM :)

**Review by lhy1024 — APPROVED** [review]

**Review by HunDunDM — APPROVED** [review]

**Pull request merged**